tests/regression/test_cli_get_regression/test_types_of_id_in_multiple_datasets_latest_.txt

```diff
@@ -1 +1 @@
-Good et al. (2019): MOHC UKESM1.0-LL model output prepared for CMIP6 ScenarioMIP ssp534-over. Version 20210205. Earth System Grid Federation. https://doi.org/10.22033/ESGF/CMIP6.6397.
+Good et al. (2019): MOHC UKESM1.0-LL model output prepared for CMIP6 ScenarioMIP. Version 20210205. Earth System Grid Federation. https://doi.org/10.22033/ESGF/CMIP6.1567.
```

Comment:
Interesting. Is something weird going on with the DOIs? It seems like they're minted at the model-MIP level, not just model level (otherwise why is ScenarioMIP in this citation?)